tests/unit/stores/UserStore.spec.js

```diff
@@ -0,0 +1,43 @@
+import { setActivePinia, createPinia } from 'pinia';
+import { useUserStore } from 'src/stores/UserStore';
+
+describe('User Store', () => {
+  beforeEach(() => {
+    // creates a fresh pinia and makes it active
+    // so it's automatically picked up by any useStore() call
+    // without having to pass it to it: `useStore(pinia)`
+    setActivePinia(createPinia());
+  });
+
+  it('should test variables in the store', () => {
```

Comment:
describe('Test store variable: username')
it('should be initialized to an empty string')
it('should return the new value after being set')

describe('Test store variable: username')
it('should be initialized to an empty string')
it('should return the new value after being set')